lib/node_modules/@stdlib/blas/ext/base/dsapxsumpw/lib/dsapxsumpw.js

```diff
@@ -39,44 +40,19 @@ var sum = require( './ndarray.js' );
 * @param {PositiveInteger} N - number of indexed elements
 * @param {number} alpha - constant
 * @param {Float32Array} x - input array
-* @param {integer} stride - stride length
+* @param {integer} strideX - stride length
 * @returns {number} sum
```

Comment:
@kgryte I noticed that in the README `alpha` is described as `scalar constant` whereas in the REPL and JSDoc it is described as `constant` while in some other packages in JSDoc it is described as `scalar`.

I suggest we use `scalar constant` to make terminology consistent within and across the packages.

Reply:
@headlessNode Agreed. "Scalar constant" seems reasonable. Feel to make those changes here and elsewhere. Thanks!